drivers/timer/stm32_lptim_timer.c

```diff
@@ -61,12 +61,33 @@ static const struct device *const clk_ctrl = DEVICE_DT_GET(STM32_CLOCK_CONTROL_N
  *    0xFFFF / (LSE freq (32768Hz) / 128)
  */
 
-static uint32_t lptim_clock_freq = KHZ(32);
-static int32_t lptim_time_base;
-
 /* The prescaler given by the DTS and to apply to the lptim_clock_freq */
 #define LPTIM_PRESCALER DT_PROP(DT_DRV_INST(0), st_prescaler)
 
+static int32_t lptim_time_base;
+static uint32_t lptim_clock_freq = CONFIG_STM32_LPTIM_CLOCK;
+static uint32_t lptim_clock_presc = LPTIM_PRESCALER;
```

Comment:
To simplify things a bit, can we get rid of LPTIM_PRESCALER, then ?

Reply:
LPTIM_PRESCALER  is used later to `LL_LPTIM_SetPrescaler`

Reply:
now lptim_clock_presc = DT_PROP(DT_DRV_INST(0), st_prescaler)